orm/models.py

```diff
@@ -161,6 +165,15 @@ def select_related(self, related):
         expr = sqlalchemy.exists(expr).select()
         return await self.database.fetch_val(expr)
 
+    def limit(self, rows_to_return: int):
+        new_query_set = self.__class__(
+            model_cls=self.model_cls,
+            filter_clauses=self.filter_clauses,
+            select_related=self._select_related,
+        )
+        new_query_set.limit_count = rows_to_return
```

Comment:
We should make sure to pass `limit_count` through in the same way that we're pasing `filter_clauses` and `select_related`. We also want to make sure that we're persisting any `limit` in `select_related` and `filter` and anywhere else that returns a new queryset.

Reply:
Right, I added limit_count to the constructor in the same way the other parameters are, and then made sure to pass limit_count through to select_related and filter. Also added a test to make sure it stays limited